docs/GPU-Windows.rst

```diff
@@ -77,9 +77,9 @@ OpenCL SDK Installation
 Installing the appropriate OpenCL SDK requires you to download the correct vendor source SDK.
 You need to know what you are going to use LightGBM!
 
--  For running on Intel, get `Intel SDK for OpenCL`_ (NOT RECOMMENDED).
+-  For running on Intel, get `Intel SDK for OpenCL`_ (**NOT RECOMMENDED**).
```

Comment:
@tpboudreau I think this is the reason why Intel OpenCL Runtime behaves oddly at Azure.

> I've run against the python package using three different versions the Intel CPU OpenCL platform. They all run successfully and give reasonable results, but for some reason they either sporadically hang at shutdown or terminate with the user script returning 'False'.

> **Warning**: using Intel OpenCL is not recommended and may crash your machine due to being non compliant to OpenCL standards.